test/integration/local/processing/get-completed-payment-requests.test.js

```diff
@@ -42,39 +45,47 @@ describe('get completed payment requests', () => {
 
   test('should not return any payment requests if none completed for agreement', async () => {
     await db.scheme.create(scheme)
-    const paymentRequests = await getCompletedPaymentRequests(paymentRequest.schemeId, paymentRequest.frn, paymentRequest.marketingYear, paymentRequest.agreementNumber, 2)
+    paymentRequest.paymentRequestNumber = 2
```

Comment:
Why do you need to set the paymentRequestNumber to 2 here?

Reply:
Because we want to change the value we're passing to the test function rather than using the default set at the start of the file